index.html

```diff
@@ -48,45 +60,41 @@ <h1 class="pb-3"><p class="pt-0 fs-4 ">Hello! Students</p>
       <h3 class="text-white pt-5 pb-3">Main Program</h3>
       <hr>
       <div class="d-flex flex-column flex-md-row">
-        <button class="btn d-flex flex-md-column ">
+        <div class="div-11 d-flex flex-md-column ">
           <i class="fa-solid fa-display fa-2xl text-white "></i>
           <h4 class="pt-2">Lecture</h4>
-          <div class="para">
-            <p class="text-white">Listen to the speakers from various countries about the messages of sharing and
-              opening.
-            </p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column ">
+          <p class="text-white ps-3 pb-2 para">Listen to the speakers from various countries about the messages of
+            sharing and
+            opening.
+          </p>
+        </div>
+        <div class="div-11 d-flex flex-md-column " >
           <i class="fa-solid fa-message fa-2xl text-white"></i>
           <h4 class="pt-2">Forum</h4>
-          <div class="para">
-            <p class="text-white ">Have the time to share your thoughts and opinions with expert for each topic.</p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column">
+          <p class="text-white para pb-2 ps-3">Have the time to share your thoughts and opinions with expert for each
+            topic.</p>
+        </div>
+        <div class="div-11 d-flex flex-md-column">
           <i class="fa-solid fa-desktop fa-2xl text-white"></i>
           <h4 class="pt-2">Workshop </h4>
-          <div class="para">
-            <p class="text-white">Trying creating your own work using open source license rather than just watching in
-              it.
-            </p>
-          </div>
-        </button>
-        <button class="btn d-flex flex-md-column ">
+          <p class="text-white ps-3 pb-2 para">Trying creating your own work using open source license rather than just
+            watching in
+            it.
+          </p>
+        </div>
+        <div class="div-11 d-flex flex-md-column ">
           <i class="fa-solid fa-code fa-2xl text-white"></i>
           <h4 class="pt-2">Challenge</h4>
-          <div class="para">
-            <p class="text-white">The questions in these coding challenges test your algorithms and data structure
-              knowledge.</p>
-          </div>
-        </button>
+          <p class="text-white ps-3 pb-2 para">The questions in these coding challenges test your algorithms and data
+            structure
+            knowledge.</p>
+        </div>
       </div>
-      <button class=" btn text-white p-3 mt-3"><u>SEE THE WHOLE PROGRAM</u></button>
+      <button class=" btn text-white p-3 pb-2 mt-3"><u>SEE THE WHOLE PROGRAM</u></button>
     </div>
```

Comment:
According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout. So kindly, on the mobile version, you should update/change the style of the `SEE THE WHOLE PROGRAM` Link to look similar to the original design 💯👌

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181493788-2db2947b-582a-4f20-adf6-41b8728a0014.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181266102-bcbcac63-9707-44de-b805-4acf0219c319.PNG" width="500" /> |